__tests__/components/RootComponent.test.js

```diff
@@ -1,3 +1,4 @@
+import 'jsdom-global/register'
```

Comment:
is this new import needed?

Reply:
Yes, actually. This has to do with the refactoring of ImportFileZone and moving the scope of `new window.FileReader()` to the component level. When the tests all run together apparently we need a test DOM from the top down RootComponent => (App) => ImportResourceTemplate => ImportFileZone in order for the test to recognize the existence of the `window` object.